Meta/Websites/man.serenityos.org/sections.inc

```diff
@@ -0,0 +1,11 @@
+<h3>Sections</h3>
+<ul>
+    <li><a href="man1/index.html">Section 1: User Programs (applets, applications, utilities, etc.)</a></li>
+    <li><a href="man2/index.html">Section 2: System Calls (getuid, mount, pledge, sendfd, etc.)</a></li>
+    <li><a href="man3/index.html">Section 3: Library Functions (basename, isatty, POSIX functions, etc.)</a></li>
+    <li><a href="man4/index.html">Section 4: Special Files (audio, mem, etc.)</a></li>
+    <li><a href="man5/index.html">Section 5: File Formats (getopt convention, Shell, SystemServer)</a></li>
+    <li><a href="man6/index.html">Section 6: Games (2048, Chess, FlappyBug, etc.)</a></li>
+    <li><a href="man7/index.html">Section 7: Miscellanea (mitigations, sys filesystem, SystemServer, etc.)</a></li>
+    <li><a href="man8/index.html">Section 8: Sysadmin Tools (dmesg, pls, sysctl, useradd, etc.)</a></li>
+</ul>
```

Comment:
(it be good to have this generated in the future!)

Reply:
I agree. That's outside the scope of this PR and will have to be done another time, but I have changed it to a markdown file so it's easier to edit!